apps/pie-docs/src/assets/styles/base/_layout.scss

```diff
@@ -65,3 +59,102 @@
         z-index: f.zIndex(low);
     }
 }
+
+// styles shared between homePage and contentPage
+.c-content {
+    flex: 1;
+    background-color: f.$color-background-default;
+    border-top: 1px solid f.$color-border-default;
+
+    & a {
+        color: f.$color-content-link-distinct;
+
+        &:visited {
+            color: f.$color-content-link-visited;
+        }
+
+        &:hover,
+        &:active,
+        &:focus {
+            text-decoration: none;
+        }
+    }
+
+    & p:has(+ .c-tokensTable) {
```

Comment:
I think you paired with @ashleynolan on this, are we happy with the browser support for`has`? I imagine for the docs site it's fine :)

Reply:
I was actually thinking of doing a small PR after this just to change it - noticed it's in a few places in pie-docs so thought it could be done separately. Agree it's probs not essential, but happy to do if it's quick and easy 😄